.github/workflows/nightly_test.yml

```diff
@@ -0,0 +1,205 @@
+name: Nightly Test Corso
+env:
+  IMAGE_NAME: ghcr.io/alcionai/corso
+on:
+  workflow_dispatch:
+
+  pull_request:
+
+  push:
+    branches: [main]
+    tags: ["v*.*.*"]
```

Comment:
Not seeing a nightly cadence here.  Just the main branch and releases.

Reply:
@meain added you to this PR.  Partly because I wouldn't mind an extra pair of eyes here.  Partly because if y'all want to move the change forward once ^ is fixed, you shouldn't wait for me.